walletconnect.sh

```diff
@@ -2,4 +2,17 @@
 
 git apply enable-walletconnect.patch
 
+npm install \
```

Comment:
I found it better to just install the dependencies (with fixed versions) than to have them in the patch, as any dependency change would cause the patch to break